base_wamas_ubl/lib/wamas/wamas2ubl.py

```diff
@@ -45,29 +46,44 @@ def get_head(self, telegram_type, key_name, head=None):
                     key,
                 )
 
-    def get_line(self, telegram_type, transfer_key_name, package_key_name=False):
+    def get_line(
+        self,
+        telegram_type,
+        transfer_key1_name,
+        transfer_key2_name=False,
+        package_key_name=False,
+    ):
         """Process a list of dict as lines of the transfers
 
         Parameters:
             telegram_type: the key to get the list out of data
-            transfer_key_name: the key in the dict that serves to identify the
+            transfer_key1_name: the key in the dict that serves to identify the
                                parent in transfers
+            transfer_key2_name: the key in the dict that serves to identify a sub-transfer
             package_key_name: the key in the dict that serves to identify the
                               related package
         """
+        transfers = {}
         if telegram_type not in self.data:
             raise ValueError("Missing telegram: %s" % telegram_type)
 
         for line in self.data[telegram_type]:
-            key = line[transfer_key_name]
+            key = line[transfer_key1_name]
             if key not in self.transfers:
                 _logger.debug(
                     "Found %s (line) record for unknown transfer, ignoring: %s",
                     telegram_type,
                     key,
                 )
                 continue
-            self.transfers[key].setdefault("lines", []).append(line)
+
+            # Create a key for the sub-transfer
+            if transfer_key2_name:
+                key = key + line[transfer_key2_name]
```

Comment:
maybe don't concatenate ?
```suggestion
                key = (key, line[transfer_key2_name])
```